frontend/src/pages/org/collection-detail.ts

```diff
@@ -108,7 +108,11 @@ export class CollectionDetail extends LiteElement {
           ${this.collection?.isPublic
             ? html`
                 <sl-tooltip content=${msg("Shareable")}>
-                  <sl-icon class="text-lg" name="people-fill"></sl-icon>
+                  <sl-icon
+                    class="text-lg"
+                    name="people-fill"
+                    style="color: var(--success)"
```

Comment:
Should be able to `text-success-600` now that tailwind is available everywhere
```suggestion
                    class="text-lg text-success-600"
                    name="people-fill"
```

Reply:
Hmm interesting.  Eventually it might be nice to use more individually scoped tokens to ensure that colour is used consistently?  (`icon-success` or something maybe, we'll see).

Should I change them all to use the tailwind color classes?

Reply:
I think `text-success` should work too?